app/views/common_parameters/_parameter.html.erb

```diff
@@ -3,15 +3,15 @@
   <table class="row">
     <tbody>
       <tr class="form-group <%= 'has-error' if f.object.errors.any? %>">
-        <% if controller_name =~ /(host|hostgroup)/ %>
```

Comment:
This change only aligns the fields with the headers. Before it, there was still an indent on the left side (about half as big) and the columns didn't align either.
Another option would be to remove this column altogether, as it doesn't really provide any useful information. 
What do you think?
It might not be worth bothering with either way since these forms will all change as part of the parameters redesign, this just caught my eye while testing my changes.